src/server/actions/Volunteers.ts

```diff
@@ -0,0 +1,32 @@
+import dbConnect from '@/utils/db-connect';
+import VolunteerSchema from '@/server/models/Volunteer';
+import { VolunteerEntity } from '@/types/dataModel/volunteer';
+
+/**
+ * Get a specific Volunteer
+ * @param volunteerId // Id of the volunteer
+ * @returns // Specific Volunteer, or null
+ */
+
+export async function getVolunteer(
+  VolunteerId: string
```

Comment:
typescript uses camel case as a variable/param naming convention, please change to `volunteerId`